ras_party/controllers/account_controller.py

```diff
@@ -726,6 +726,10 @@ def put_email_verification(token, tran, session):
 
         if respondent:
             update_verified_email_address(respondent, tran)
+            if not respondent.pending_email_address and respondent.password_verification_token:
```

Comment:
This still needs some work, to get to this line we know this is a pending email update. The respondent has received an email which when they clicks updates the email with the pending and then removes the pending_email (line 798) (i.e update_verified_email_address) 

Now because of that and the order this has been put in ```if not respondent.pending_email_address``` becomes redundant as the code underneath will always set respondent.pending_email_address to None, all we need to do is check the verification token existence

Reply:
This has been updated